homeassistant/components/heatzy/switch.py

```diff
@@ -0,0 +1,79 @@
+"""Sensors for Heatzy."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from heatzypy.exception import HeatzyException
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import HeatzyDataUpdateCoordinator
+from .const import CONF_ALIAS, CONF_ATTR, CONF_ATTRS, CONF_LOCK, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Set the sensor platform."""
+    coordinator = hass.data[DOMAIN][entry.entry_id]
+    entities = [
+        LockSwitchEntity(coordinator, unique_id)
+        for unique_id in coordinator.data.keys()
+    ]
+    async_add_entities(entities)
+
+
+class LockSwitchEntity(CoordinatorEntity[HeatzyDataUpdateCoordinator], SwitchEntity):
+    """Lock Switch."""
```

Comment:
What is a Lock Switch? Is it something Heatzy specific? Should that be documented?